roles/packages_installation/defaults/main.yml

```diff
@@ -17,26 +17,76 @@ packages:
     - python3-lxml
     - python3-netaddr
     - python3-passlib
-  opensuse:
+  leap15:
+    - gcc
     - git
-    - libvirt-client
-    - qemu-kvm
+    - guestfs-tools
+    - libguestfs
     - libvirt
-    - python311-pip
-    - python311-devel
+    - libvirt-client
+    - libvirt-devel
+    - mkisofs
     - pkg-config
+    - podman
+    - python311-devel
+    - python311-libvirt-python
+    - python311-lxml
+    - python311-netaddr
+    - python311-passlib
+    - python311-pip
+    - qemu
+    - qemu-kvm
+    - sshpass
+    - virt-install
+  leap16:
     - gcc
+    - git-core
+    - guestfs-tools
+    - libguestfs
+    - libvirt
+    - libvirt-daemon
+    - libvirt-dbus
+    - libvirt-client
     - libvirt-devel
+    - libvirt-daemon-qemu
+    - libvirt-daemon-config-nwfilter
     - mkisofs
+    - pkg-config
+    - podman
+    - python313-devel
+    - python313-lxml
+    - python313-netaddr
+    - python313-passlib
+    - python313-pip
```

Comment:
I guess we also need `python313-libvirt-python` here?

Reply:
yes, fixed!